privacy/privacy-policy.md

```diff
@@ -2,14 +2,16 @@
 
 # Basecamp privacy policy
 
-*Last updated: May 24, 2018*
+*Last updated: December 20, 2018*
 
 The privacy of your data — and it is your data, not ours! — is a big deal to us. We’ll only ever access your account to help you with a problem or squash a software bug. We’ll never open any uploaded files unless you ask us to. We log all access to all accounts by IP address, so we can always verify that no unauthorized access has happened for as long as the logs are kept.
 
 ## Identity & access
 
 When you sign up for Basecamp, we ask for your name, company name, and email address. That's just so you can personalize your new account, and we can send you invoices, updates, or other essential information. We’ll never sell your personal info to third parties, and we won’t use your name or company in marketing statements without your permission, either.
 
+When you pay for Basecamp, we'll ask for your credit card and billing address. That's so we can charge you for service, calculate taxes due, and send you invoices. Your credit card is passed directly to our payment processor; we don't store it, and it doesn't pass through our servers at all. We store your billing address to print on your invoices, to calculate any sales tax due in the United States, and to detect fraudulent credit card transactions.
```

Comment:
"... we don't store it..." <-- "we don't store your card number"

I'd also consider putting the "print on your invoices" bit as the last listed rationale -- it's pretty low priority compared to the other two reasons.

Reply:
We do store last 4 digits of the card though. Is that worth mentioning?